pkg/tfbridge/provider.go

```diff
@@ -409,16 +415,16 @@ func validateProviderConfig(ctx context.Context, p *Provider, config shim.Resour
 	// Perform validation of the config state so we can offer nice errors.
 	warns, errs := p.tf.Validate(config)
 	for _, warn := range warns {
-		if err := p.host.Log(ctx, diag.Warning, "", fmt.Sprintf("provider config warning: %v", warn)); err != nil {
-			return nil, err
+		logErr := p.host.Log(ctx, diag.Warning, "", fmt.Sprintf("provider config warning: %v", warn))
+		if logErr != nil {
+			glog.V(9).Infof("Failed to log to the engine: %v", logErr)
+			return nil, nil
 		}
 	}
 
-	if len(errs) > 0 {
-		return nil, errors.Wrap(multierror.Append(nil, errs...), "could not validate provider configuration")
-	}
-
-	return nil, nil
+	failures := p.formatCheckFailures(urn, true /*isProvider*/, []string{"provider"}, p.config,
```

Comment:
Hmm instead of "provider" we can extract the URN name here to narrow it down better.